packages/google-oauth/package.js

```diff
@@ -0,0 +1,17 @@
+Package.describe({
+  summary: "Google OAuth flow",
+  version: "1.2.0"
+});
+
+Package.onUse(function(api) {
+  api.use('oauth2', ['client', 'server']);
+  api.use('oauth', ['client', 'server']);
+  api.use('http', ['server']);
+  api.use(['underscore', 'service-configuration'], ['client', 'server']);
+  api.use(['random', 'templating@1.2.13'], 'client');
```

Comment:
Is there a particular reason we need to depend on not only `templating` but specifically `templating@1.2.13`?  The `facebook-oauth` package doesn't use `templating` at all.